monitor/services/src/main/java/org/datacleaner/monitor/server/SchedulingServiceImpl.java

```diff
@@ -644,13 +644,20 @@ public ExecutionLog getLatestExecution(TenantIdentifier tenant, JobIdentifier jo
                 new Func<RepositoryFile, ExecutionIdentifier>() {
                     @Override
                     public ExecutionIdentifier eval(final RepositoryFile file) {
-                        ExecutionIdentifier result = file.readFile(new Func<InputStream, ExecutionIdentifier>() {
-                            @Override
-                            public ExecutionIdentifier eval(InputStream in) {
-                                return SaxExecutionIdentifierReader.read(in, file.getQualifiedPath());
-                            }
-                        });
-                        return result;
+                        try {
+                            ExecutionIdentifier result = file.readFile(new Func<InputStream, ExecutionIdentifier>() {
+                                @Override
+                                public ExecutionIdentifier eval(InputStream in) {
+                                    return SaxExecutionIdentifierReader.read(in, file.getQualifiedPath());
+                                }
+                            });
+                            return result;
+                        } catch (Exception e) {
+                            logger.warn("The file " + file.getQualifiedPath() + " could not be read or parsed correctly"
+                                    + e);
```

Comment:
Hmmm, this one was missed. We should probably start using the review feature (I've also been resisting though, but that would avoid missing stuff)